common/meta/raft-store/src/state_machine/database_lookup.rs

```diff
@@ -0,0 +1,85 @@
+use std::fmt;
+use std::io::Cursor;
+
+use bytes::BytesMut;
+use common_exception::ErrorCode;
+use common_io::prelude::BinaryRead;
+use common_io::prelude::BinaryWriteBuf;
+use common_meta_sled_store::sled::IVec;
+use common_meta_sled_store::SledOrderedSerde;
+use serde::Deserialize;
+use serde::Serialize;
+
+const DB_LOOKUP_KEY_DELIMITER: u8 = '/' as u8;
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct DatabaseLookupKey {
+    tenant_id: String,
+    delimiter: u8,
+    database_name: String,
+}
+
+impl DatabaseLookupKey {
+    pub fn new(tenant_id: String, database_name: String) -> Self {
```

Comment:
It would be more appropriate to change the all `tenant_id` to `tenant`, keeping them consistent with query :)